### PR TITLE
Lokalise: Translations update

### PR DESCRIPTION
**Pull request by appinteractive**
This pull request was initiated by Lokalise (user Mogge) at 2019-11-17 17:47:59

### CODE REVIEW

**Comment by Mogge** [comment]
This PR was triggered by Lokalize as indicated in the picture. To see this screen you have to click on the Download icon.

grafik

File: locale/fr.json [hunk 1]
```diff
@@ -0,0 +1,2122 @@
...
+        "login": {
+            "translation": "Connexion",
+            "notes": ""
+        },
+        "logout": {
+            "translation": "Déconnexion",
+            "notes": ""
+        },
+        "email": {
+            "translation": "Votre courriel",
```
Comment (Elweyn) [inline locale/fr.json]: "Votre courriel" seems a bit strange, I think it should be "Votre mail" or "Votre addresse mail"

File: locale/fr.json [hunk 2]
```diff
@@ -0,0 +1,2122 @@
...
+        "moreInfoHint": {
+            "translation": "à la page de présentation",
+            "notes": ""
+        },
+        "success": {
+            "translation": "Vous êtes connecté!",
+            "notes": ""
+        },
+        "failure": {
+            "translation": "Adresse e-mail ou mot de passe incorrect.",
```
Comment (Elweyn) [inline locale/fr.json]: Changed 'e-mail' to 'mail' on localise

File: locale/fr.json [hunk 3]
```diff
@@ -0,0 +1,2122 @@
...
+        "following": {
+            "translation": "Suivant",
+            "notes": ""
+        },
+        "shouted": {
+            "translation": "Recommandé",
+            "notes": ""
+        },
+        "commented": {
+            "translation": "Comentado",
```
Comment (Elweyn) [inline locale/fr.json]: Need a new pull from localise, has been changed

**Comment by Mogge** [comment]
@roschaefer this is fixed:
grafik

**Comment by roschaefer** [comment]
@Mogge @Elweyn alright, so I'm going to close this? We can trigger another PR from lokalise any time.

Funny by the way that @appinteractive is the PR creator.. :thinking:

**Comment by Mogge** [comment]
Yes, please close. We should establish an import from lokalise once a week.

**Pull request closed without merging**